vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/implementation/CsvFileDataSource.java

```diff
@@ -73,4 +73,9 @@ public String toString() {
 		return "CsvFileDataSource [csvFile=" + getFile() + "]";
 	}
 
+	@Override
+	public String getSyntacticRepresentation() {
+		return "load-csv(\"" + getFile() + "\") .";
+	}
+
```

Comment:
don't we want the toString() of DataSources to return getSyntacticRepresentation()?
@mkroetzsch what do you think?

Reply:
I was told by Markus to leave this as it is.